odl/solvers/functional/functional.py

```diff
@@ -884,3 +885,44 @@ def convex_conj(self):
         """
         return self.functional.convex_conj.translated(
             self.linear_term)
+
+
+class FunctionalDefaultConvexConjugate(Functional):
+
+    """The ``Functional`` representing ``f^*(.)``.
+
+    Does not provide a way in which to evaluate the functional, intended to
+    be used for its `proximal`.
+    """
+
+    def __init__(self, func):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        func : `Functional`
+            Function corresponding to F.
```

Comment:
Functional <-> Function. Settle for one (Functional). Also `F` here and `f^*` above, both don't mean anything.